resources/platformsh/ibexa-oss/3.3.x-dev/.platform.app.yaml

```diff
@@ -103,6 +103,9 @@ mounts:
     'public/var':
         source: local
         source_path: var
+    'src/Migrations/Ibexa':
```

Comment:
Not really useful in ibexa-oss because Migration Bundle is not available for open source